engine/src/main/java/org/terasology/engine/rendering/dag/ModuleRendering.java

```diff
@@ -78,8 +78,8 @@ public void setInitPriority(int initPriority) {
     }
 
     protected void setProvidingModule(Class implementingClass) {
-        ModuleManager moduleManager = context.get(ModuleManager.class);
-        this.providingModule = moduleManager.getEnvironment().getModuleProviding(implementingClass);
+        ModuleManager manager = context.get(ModuleManager.class);
```

Comment:
Why can't we use the member field here? It is initialized exactly the same way, so this should be unnecessary.